### PR TITLE
Fb/location page

### PR DESCRIPTION
**Pull request by surajprakashpatel**

### CODE REVIEW

File: app/src/main/java/com/zariya/zariya/auth/presentation/fragment/LocationFragment.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            override fun afterTextChanged(s: Editable) {
+                val search = s.toString()
+
+            }
+        });
+    }
+
+    private fun showCitiesList() {
+
+        val layoutManager = GridLayoutManager(activity, 4)
```
Comment (VivekSharma811) [inline app/src/main/java/com/zariya/zariya/auth/presentation/fragment/LocationFragment.kt:72]: assign layoutmanager is xml instead

File: app/src/main/java/com/zariya/zariya/auth/presentation/fragment/LocationFragment.kt [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        binding.rvPopularCities.setLayoutManager(layoutManager)
+        popularCitiesAdapter =  PopularCitiesAdapter(cityNames,cityImages)
+        binding.rvPopularCities.setAdapter(popularCitiesAdapter)
+
+
+        binding.rvOtherCities.setLayoutManager(LinearLayoutManager(context))
```
Comment (VivekSharma811) [inline app/src/main/java/com/zariya/zariya/auth/presentation/fragment/LocationFragment.kt:78]: assign layoutmanager is xml instead

File: app/src/main/res/layout/fragment_location.xml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".auth.presentation.fragment.LocationFragment"
+    android:background="@color/yellow_background">
+
+    <androidx.appcompat.widget.LinearLayoutCompat
```
Comment (VivekSharma811) [inline app/src/main/res/layout/fragment_location.xml:11]: Use only constraint layout

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -25,4 +25,23 @@
...
     <string name="google">Google</string>
 
+
+    <string name="pick_a_region">Pick A Region</string>
+    <string name="search_for_your_city">Search for your city</string>
+    <string name="auto_detect_my_location">Auto Detect My Location</string>
+
+
+    <!--    Popular Cities-->
+    <string name="delhi">Delhi</string>
```
Comment (VivekSharma811) [inline app/src/main/res/values/strings.xml:35]: move them to a string array

**Review by VivekSharma811 — APPROVED** [review]

**Review by Mahima081 — APPROVED** [review]

**Pull request merged**